### ISSUE TITLE
console.log scattered throughout source

### ISSUE THREAD

**Issue by mg1075** [cdjackson/angular-dygraphs#8]
The source has calls to console.log scattered throughout which is nice for development but not for production.

**Comment by cdjackson** [comment]
I agree this can probably be removed or reduced, but you can also remove it for production (which is what I do here).
